nexus/tests/integration_tests/endpoints.rs

```diff
@@ -579,6 +579,15 @@ lazy_static! {
                 )
             ],
         },
+
+        VerifyEndpoint {
+            url: "/by-id/organizations/{id}",
```

Comment:
Note the `{id}` is important here. That's the only way the id will get substituted.